src/mmrelay/message_queue.py

```
@@ -143,80 +180,185 @@ def start(self, message_delay: float = DEFAULT_MESSAGE_DELAY) -> None:
                         daemon=True,
                     ).start()
-                else:
-                    _shutdown(self._executor)
-                self._executor = None
+            else:
+                task.cancel()
+                try:
+                    task_loop.run_until_complete(task)
+                except (asyncio.CancelledError, RuntimeError):
+                    pass
+                except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)